RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Engines.cfg

```diff
@@ -3327,3 +3327,309 @@
 	fx_exhaustFlame_yellow_tiny = 0.0, -0.17, 0.163, 0.0, 1.0, 0.0, running
 
 }
++PART[liquidEngine3]:FOR[RealismOverhaul]:NEEDS[!RftS,!RealFuels_StockEngines]
+{
+	@name = RO_KVD1
+	%RSSROConfig = True
+	!MODULE[TweakScale]
+	{
+	}
+	@MODEL
+	{
+		@model = Squad/Parts/Engine/liquidEngineLV-909/model
+		%scale = 2.8, 4.0, 2.8
+	}
+	@MODEL:NEEDS[VenStockRevamp]
+	{
+		@model = VenStockRevamp/Squad/Parts/Engine/LV909B
+	}
+	MODEL
+	{
+		model = RealismOverhaul/Models/KVDvernStock
+		texture = ksp_r_microEngine_diff, Squad/Parts/Engine/liquidEngineLV-1R/ksp_r_microEngine_diff
+		scale = 1, 1, 1
+		position = 0, -1.15, -0.585
+		rotation = 16.5, 0, 0
+	}
+	MODEL
+	{
+		model = RealismOverhaul/Models/KVDvernStock
+		texture = ksp_r_microEngine_diff, Squad/Parts/Engine/liquidEngineLV-1R/ksp_r_microEngine_diff
+		scale = 1, 1, 1
+		position = 0, -1.15, 0.585
+		rotation = 16.5, 180, 0
+	}
+	@MODEL,1:NEEDS[VenStockRevamp]		//Hex-editted mu files originally by Ven, re-distributed under CC-BY-4.0
+	{
+		@model = RealismOverhaul/Models/KVDvernVSR
+		!texture = DELETE
+		texture = SmallEngines_CLR, VenStockRevamp/Squad/Parts/Engine/SmallEngines_CLR
+		texture = SmallEngines_NRM, VenStockRevamp/Squad/Parts/Engine/SmallEngines_NRM
+	}
+	@MODEL,2:NEEDS[VenStockRevamp]
+	{
+		@model = RealismOverhaul/Models/KVDvernVSR
+		//!texture = DELETE
+		texture = SmallEngines_CLR, VenStockRevamp/Squad/Parts/Engine/SmallEngines_CLR
+		texture = SmallEngines_NRM, VenStockRevamp/Squad/Parts/Engine/SmallEngines_NRM
+	}
+	MODEL
+	{
+		model = Squad/Parts/Structural/strutOcto/model
+		scale = 3, 4.5, 3
+		position = 0, 0.35, 0
+		rotation = 0, 22.5, 0
+	}
+
+	@scale = 1.0
+	%rescaleFactor = 1.0
+	%node_stack_top = 0.0, 0.9, 0.0, 0.0, 1.0, 0.0, 2
+	%node_stack_shroud = 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0
+	%node_stack_bottom = 0.0, -1.513004, 0.0, 0.0, 1.0, 0.0, 2
+	%title = KVD-1/CE-7.5 Vacuum Engine
+	%manufacturer = KB Khimavtomatiki (Kosberg) / ISRO
+	%description = Staged combustion hydrolox upper stage engine intended for use on the N-1M and considered for use on Proton and Angara, eventually licensed to India for its GSLV MkI and MkII.  Later versions were developed by India for domestic use on the as the CE-7.5.  The main engine bell is fixed in place, and two verniers are used to provide combined pitch-yaw-roll control; this leads to lower control authority than on stages where the main engine can gimbal.  This engine runs with a somewhat higher than average O/F ratio, resulting in a denser than average hydrolox stage.
+	%attachRules = 1,1,1,1,0
+	%mass = 0.282
+	%maxTemp = 1700
+
+
+!fx_exhaustFlame_blue_small = DELETE
+!fx_exhaustLight_blue = DELETE
+!fx_smokeTrail_light = DELETE
+
+
+!sound_vent_medium = DELETE
+!sound_rocket_hard = DELETE
+!sound_vent_soft = DELETE
+!sound_explosion_low = DELETE
+
+EFFECTS
+{
+  running
+  {
+    AUDIO
+    {
+      channel = Ship
+      clip = sound_rocket_spurts
+      volume = 0.0 0.0
```

Comment:
That only applies to direct throttle effects.  Running effects use thrust/max thrust as their input value.